grails-app/domain/org/pih/warehouse/inventory/OutboundStockMovementListItem.groovy

```diff
@@ -108,6 +108,7 @@ class OutboundStockMovementListItem implements Serializable {
                 name                : name,
                 description         : description,
                 statusCode          : statusCode?.toString(),
+                statusVariant       : status?.variant?.name,
```

Comment:
The variant here is an indicator that can be as an example: "warning", "success", etc. It is represented as a colorful dot on our list pages:
![image](https://github.com/openboxes/openboxes/assets/83239466/2f897126-a77d-4ccf-9180-45dda6716a86)

I had to move it here, because of the implementation that we had. This implementation doesn't allow us to have different options on the filtering dropdown, than the statuses on the stock movement list. In this ticket I have to remove "Verified" from the filtering when the location supports "request approval", but on the current list we can have "verified" status